packages/snap/locales/fr.json

```diff
@@ -15,19 +17,20 @@
     "one": "Il y a {count} activation LXP active"
   },
   "lxpAddress": {
-    "heading": "Adresse du portefeuille Linea XP",
+    "heading": "Adresse du portefeuille Linea LXP",
```

Comment:
Also wrong and should be reverted, it's Linea XP. Or is it really called Linea LXP now???

Reply:
Same